content/docs/learning-guide/tutorials/assets/mesh-assets.md

```diff
@@ -0,0 +1,67 @@
+---
+linkTitle: Mesh Processing
+title: Customize Mesh Asset Processing
+description: Learn to customize how mesh assets are processed for Open 3D Engine (O3DE) with Scene Settings.
+weight: 100
+toc: true
+---
+
+This tutorial demonstrates how to customize mesh asset processing for **Open 3D Engine (O3DE)**. You can use any mesh for this tutorial. The primary 3D scene format supported by O3DE is `.fbx`, so it's recommended to use a simple mesh asset saved to a `.fbx` file for this tutorial. If you don't have your own source asset, you can use one of the assets provided with O3DE.
+
+## Preparation
+
+**Asset Processor** is an application and a service that runs in the background while you use O3DE. Asset Processor automatically detects source assets that are placed in scan directories and schedules process jobs for the found source assets. The process jobs generate product assets that are runtime optimized for O3DE. The end-to-end process from source asset to product asset is the **Asset Pipeline**. To learn more about Asset Processor and the Asset Pipeline, refer to the [Assets](/docs/user-guide/assets/) topic in the User Guide.
+
+Asset Processor uses *scan directories* to find assets that need to be processed. Your entire project directory structure is a scan directory. Place your `.fbx` source asset somewhere in your project, such as the `Assets` subdirectory, so it can be detected by Asset Processor.
+
+## Processing a mesh asset
+
+When you placed your `.fbx` source asset in your project, Asset processor detected the asset and processed it with a default set of rules for `.fbx` source assets. To customize the rules that process a mesh asset, follow the steps below:
+
+1. In **O3DE Editor** locate your asset in **Asset Browser**. If you don't have an asset of your own, you can type `fbx` into the search field at the top of asset browser and use one of the provided `.fbx` files such as `sphere.fbx`.

```

Comment:
Minor issue - You suggested `sphere.fbx`, but the examples used something else. You might not be able to suggest users use `o3de-logo.fbx` if its not a standard asset, but you can call out that you're using it for your examples. 
```suggestion
1. In **O3DE Editor** locate your asset in **Asset Browser**. If you don't have an asset of your own, you can type `fbx` into the search field at the top of asset browser and use one of the provided `.fbx` files. In this example, we'll use `o3de-logo.fbx`.

```

Reply:
The readers won't have access to that file. I couldn't get the Gem into the repo in time for this doc. I suggest the reader use their own file or the one of the included files. I am using that file specifically because I need a complex mesh for the PhysX collider tutorial to demonstrate decomposition and physmaterials and I don't want there to be any IP issues wit the documentation. We may get that file out eventually.